addons/website_forum/tests/test_performance.py

```diff
@@ -0,0 +1,71 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import random
+
+from odoo.addons.website.tests.test_performance import UtilPerf
+
+
+class TestForumPerformance(UtilPerf):
+    def setUp(self):
+        super().setUp()
+        self.user_employee = self.env['res.users'].with_context({'no_reset_password': True}).create({
```

Comment:
Better use the ``mail_new_test_user`` helper (sets all context keys necessary to speedup user creation)